crates/adapters/Cargo.toml

```diff
@@ -136,23 +136,23 @@ jemalloc_pprof = "0.1.0"
 psutil = "3.2.2"
 
 [dev-dependencies]
-actix-test = "0.1.1"
-bstr = { version = "0.2.1", features = ["serde1"] }
-serde_json = "1.0.127"
+actix-test = { workspace = true }
+bstr = { workspace = true, features = ["serde1"] }
+serde_json = { workspace = true }
 size-of = { version = "0.1.5", package = "feldera-size-of", features = ["time-std", "ordered-float"] }
-tempfile = "3.3.0"
-proptest = "1.5.0"
-proptest-derive = "0.5.0"
-futures = "0.3.30"
-bytestring = "1.2.0"
-actix-codec = "0.5.0"
-async-stream = "0.3.5"
-futures-timer = "3.0.2"
-test_bin = "0.4.0"
-reqwest = { version = "0.11.20", features = ["blocking"] }
-serial_test = "2.0.0"
-rust_decimal_macros = "1.32"
-mockall = "0.12.1"
-pretty_assertions = "1.4.0"
-feldera-sqllib = { path = "../sqllib" }
-google-cloud-googleapis = "0.15.0"
+tempfile = { workspace = true }
+proptest = { workspace = true }
+proptest-derive = { workspace = true }
+futures = { workspace = true }
+bytestring = { workspace = true }
+actix-codec = { workspace = true }
+async-stream = { workspace = true }
+futures-timer = { workspace = true }
+test_bin = { workspace = true }
+reqwest = { workspace = true, features = ["blocking"] }
+serial_test = { workspace = true }
+rust_decimal_macros = { workspace = true }
+mockall = { workspace = true }
+pretty_assertions = { workspace = true }
+feldera-sqllib = { workspace = true }
+google-cloud-googleapis = { workspace = true }
```

Comment:
look how long this list is... no wonder we take forever to compile 
normally we should only link the adapters we need

Reply:
dependencies normally are only compiled once and re-used